docs/01-overview/02-main-areas/application-connectivity/ac-07-application-connector-certificates.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Application Connector Certificates
```

Comment:
Isn't it too detaild for the Main areas section?

Reply:
Moved the part on how to provide a custom cert and key to Operations.